packages/amplify-cli/src/__tests__/commands/upgrade.pkg.test.ts

```diff
@@ -98,13 +109,14 @@ describe('run upgrade using packaged CLI', () => {
     // validate
     expect(fs_mock.move.mock.calls[0]).toMatchInlineSnapshot(`
       Array [
-        "homedir/bin/amplify-pkg-linux",
-        "homedir/bin/amplify",
+        "${path.posix.join('homedir', 'bin', 'amplify-pkg-linux')}",
```

Comment:
If the new serializer is working properly, we shouldn't need the `posix` version anywhere - just `path.join()` should work.